agent/rpm/targets.json

```diff
@@ -0,0 +1,11 @@
+{
+    "targets": {
+        "python3": {
+            "condition": "0%{?rhel} != 8"
+        },
+        "python39": {
+            "condition": "0%{?rhel} == 8"
+        }
+    }
+}
+
```

Comment:
Is there an intention to add more than `"condition"` in each object, or should this be simplified to `"python3": "0%{?rhel} != 0"` & etc?

Reply:
At this point, I don't think we need to add anything, but I don't know for sure.

Reply:
I simplified it as you suggested. We can always complicate it later if necessary.

Reply:
@ndokos, using a loop to iterate over mutually-exclusive targets seems like a fairly heavy mechanism for what it accomplishes.  Unless we have some future expansion in mind, I think we would be better served by a straightforward conditional definition of `tname` to either `python3`/`python39`.

I assume I'll be modifying this file soon, would you be OK with me making that change?

Reply:
Looking closer, I think this definition could be made along side the definition of `__python` which would be pretty simple.